test/blackbox-tests/test-cases/coq/github3624.t

```diff
@@ -16,5 +16,5 @@ that the error message is good when the coq extension is not enabled.
   1 | (coq.theory
   2 |  (name foo))
   Error: 'coq.theory' is available only when coq is enabled in the dune-project
-  file. You must enable it using (using coq 0.5) in your dune-project file.
+  file. You must enable it using (using coq 0.6) in your dune-project file.
```

Comment:
```suggestion
  file. You must enable it using (using ``coq 0.6``) in your ``dune-project`` file.
```

Reply:
Autogenerated output.

Reply:
Autogenerated from where? Like by `odoc`?

Reply:
This is the command line error message of Dune.